.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-    "endOfLine" : "lf",
+    "endOfLine": "auto",
```

Comment:
this will make the line endings dependant on the OS, rather than fixed for all environemnts